docs/B-Computations/style-guidelines.md

```diff
@@ -1,5 +1,8 @@
 ---
 sidebar_position: 5
+title: "Style-Guidelines"
```

Comment:
```suggestion
title: Style Guidelines
```
The `title` property in frontmatter shows up in the side bar. Consider keeping it consistent
![image](https://user-images.githubusercontent.com/32626950/142272048-75875bce-7b4d-49a8-a407-4fe3e083ebab.png)

Reply:
Thank you for the feedback, I am making this change to my latest commit. I'm surprised I did not catch that 👍